pypeit/spectrographs/keck_nirspec.py

```diff
@@ -6,24 +7,1124 @@
+        par['calibrations']['slitedges']['max_shift_adj'] = 0.5
+        par['calibrations']['slitedges']['trace_thresh'] = 10.
+        par['calibrations']['slitedges']['left_right_pca'] = True
+        par['calibrations']['slitedges']['length_range'] = 0.3
+        par['calibrations']['slitedges']['max_nudge'] = 10.
+        par['calibrations']['slitedges']['overlap'] = True
+        par['calibrations']['slitedges']['dlength_range'] = 0.1
+
+        # Flats
+        par['calibrations']['flatfield']['tweak_slits_thresh'] = 0.80
+
+        # Extraction
+        par['reduce']['skysub']['bspline_spacing'] = 0.8
+        par['reduce']['extraction']['sn_gauss'] = 4.0
+        par['reduce']['findobj']['fof_link'] = 0.2
+
+        # Flexure
+        par['flexure']['spec_method'] = 'skip'
+
+        par['scienceframe']['process']['sigclip'] = 20.0
+        par['scienceframe']['process']['satpix'] ='nothing'
+
+        # Should be we be illumflattening?
+
+        # Flats
+        turn_off = dict(use_illumflat=False, use_biasimage=False, use_overscan=False,
+                        use_darkimage=False, use_pixelflat=False)
+        par.reset_all_processimages_par(**turn_off)
+
+        #turn_off = dict(use_biasimage=False, use_overscan=False)
+        #par.reset_all_processimages_par(**turn_off)
+
+        # Specify if cleaning cosmic ray hits/bad pixels
+
+        # The settings below enable NIRSPEC dark subtraction from the
+        # traceframe and pixelflatframe, but enforce that this bias won't be
+        # subtracted from other images. It is a hack for now, because
+        # eventually we want to perform this operation with the dark frame
+        # class, and we want to attach individual sets of darks to specific
+        # images.
+        #par['calibrations']['biasframe']['useframe'] = 'bias'
+        #par['calibrations']['traceframe']['process']['bias'] = 'force'
+        #par['calibrations']['pixelflatframe']['process']['bias'] = 'force'
+        #par['calibrations']['arcframe']['process']['bias'] = 'skip'
+        #par['calibrations']['tiltframe']['process']['bias'] = 'skip'
+        #par['calibrations']['standardframe']['process']['bias'] = 'skip'
+        #par['scienceframe']['process']['bias'] = 'skip'
+
+        # Set the default exposure time ranges for the frame typing
+        par['calibrations']['standardframe']['exprng'] = [None, None]
+        par['calibrations']['arcframe']['exprng'] = [20, None]
+        par['calibrations']['darkframe']['exprng'] = [None, None]
+        par['scienceframe']['exprng'] = [None, None]
+
+        # Sensitivity function parameters
+        par['sensfunc']['algorithm'] = 'IR'
+        par['sensfunc']['polyorder'] = 8
+        #par['sensfunc']['IR']['telgridfile'] = 'TelFit_MaunaKea_3100_26100_R20000.fits'
+        par['sensfunc']['IR']['telgridfile'] = '/Users/asc/OneDriveDocs/Caltech/KVSP2023/PypeItDev/testTelluricRemoval/TellPCA_9300_55100_R60000.fits'
```

Comment:
replace this with `TellPCA_9300_55100_R60000.fits`

Reply:
fixed